bluemira/builders/EUDEMO/reactor.py

```diff
@@ -282,14 +282,16 @@ def build_first_wall(self, component_tree: Component, **kwargs):
         config = self._process_design_stage_config(name, default_config)
 
         plasma = component_tree.get_component(self.PLASMA)
-        builder = FirstWallBuilder(
+        builder = InVesselComponentBuilder(
             self._params.to_dict(), build_config=config, equilibrium=plasma.equilibrium
         )
         self.register_builder(builder, name)
 
         component = super()._build_stage(name)
         bluemira_print(f"Completed design stage: {name}")
-        return component
+        # TODO(hsaunders1904): this is not functional at the moment,
+        # need to check parameter access is correct
+        return build_ivc_xz_shapes(component, self._params.get_param("c_rm").value)
```

Comment:
I normally do `self._params.c_rm.value` not sure if that would work at the reactor level